diff --git a/crate1/Cargo.toml b/crate1/Cargo.toml
--- a/crate1/Cargo.toml
+++ b/crate1/Cargo.toml
@@ -3,4 +3,4 @@
 version = "0.1.0"
 
 [dependencies]
-serde = { workspace = true, optional = true }
+serde = { version = "1", features = ["derive"], optional = true }

diff --git a/Cargo.toml b/Cargo.toml
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -3,4 +3,3 @@
 resolver = "2"
 
 [workspace.dependencies]
-serde = { version = "1", features = ["derive"] }


